tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -66,6 +68,25 @@ impl<T: Numeric> CudaVec<T> {
     pub fn is_empty(&self) -> bool {
         self.len == 0
     }
+
+    pub(crate) fn range_bounds_to_start_end<R>(&self, range: R) -> std::ops::RangeInclusive<usize>
+    where
+        R: std::ops::RangeBounds<usize>,
+    {
+        let start = match range.start_bound() {
+            Unbounded => 0usize,
+            Included(start) => *start,
+            Excluded(start) => *start + 1,
+        };
+
+        let end = match range.end_bound() {
+            Unbounded => saturating_sub(self.len(), 1),
+            Included(end) => *end,
+            Excluded(end) => saturating_sub(*end, 1),
```

Comment:
same here and you can then remove the import otherwise clippy will be annoying 🙂